src/Operators/finitedifference.jl

```diff
@@ -873,7 +878,20 @@ U(v,x)[i] = \\begin{cases}
 This stencil is based on [WickerSkamarock2002](@cite), eq. 4(a).
 
 Supported boundary conditions are:
-- At the moment this is defined only for periodic meshes.
+- [`SetValue(x₀)`](@ref): set the value of `x` to be `x₀` on the boundary. On
+  the left boundary the stencil is
+  ```math
+  U(v,x)[\\tfrac{1}{2}] = \\begin{cases}
+   v[\\tfrac{1}{2}] \\left(8 x_0 + 4 x[1+\\tfrac{1}{2}] \\right) / 12  \\textrm{, if } v[i] > 0 \\\\
+   v[\\tfrac{1}{2}] \\left(4 x_0 + 10 x[1+\\tfrac{1}{2}] -2 x[1+\\tfrac{3}{2}]  \\right) / 12  \\textrm{, if } v[i] < 0
```

Comment:
Not sure if the docs here are a good representation of the code or if this is more aspirational